packages/cmk-agent-based/cmk/agent_based/v2/_get_average.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+# Copyright (C) 2019 Checkmk GmbH - License: GNU General Public License v2
+# This file is part of Checkmk (https://checkmk.com). It is subject to the terms and
+# conditions defined in the file COPYING, which is part of this source code package.
+
+from collections.abc import MutableMapping
+from typing import Any, cast
+
+def get_average(  # type: ignore[misc]
+    value_store: MutableMapping[str, Any],
```

Comment:
I think we can use object here? I'd actually prefer to tell the truth here as well...